README.md

```diff
@@ -58,6 +58,33 @@ the following environment variables to your container:
   - `DB_USER` (default: `root`)
   - `DB_PASS`
 
```

Comment:
please add `DB_DRIVER` here and describe under which conditions it's set implicitly and when it needs to be set explicitly.
for usage with external hosts, the default must be mysql in order to provide backward-compatibility.